src/config.c

```diff
@@ -760,16 +762,19 @@ void configSetCommand(client *c) {
                     }
                 }
                 /* Apply function not stored, store it */
-                if (!exists)
+                if (!exists) {
                     apply_fns[j] = set_configs[i]->interface.apply;
+                    config_map_fns[j] = i;
```

Comment:
This fix isn't perfect because in case you have multiple configs with the same apply function you will only report the first one as the cause of the error, while the other one or both are actually the cause.
Take for example an attempt to apply both `tls-cert-file` and `tls-key-file` together:
`CONFIG set tls-cert-file some_existing_file.pem tls-key-file some-missing-file.pem`
I think that if `tls-key-file` doesn't exits it'll report an error due to `tls-cert-file`. 

Ideally we should report all input configuration arguments that are mapped to the failed apply function. On the other hand it might not make much sense to complicated the code for the sake of error reporting so maybe just report all the input configuration parameters or none of them? WDYT?

Reply:
i also think we can live with that, don't wanna complicate the code much more (certainly not for a log message).

so ok to be merged?

Reply:
So lets rephrase the message to indicate this config **might** be the culprit.